website/content/docs/job-specification/disconnect.mdx

```diff
@@ -0,0 +1,202 @@
+---
+layout: docs
+page_title: disconnect Block - Job Specification
+description: |-
+  The "disconnect" block describes the behaviuor of both the Nomad server and 
+  client in case of a network partition, as well as how to reconcile the workloads
+  in case of a reconnection.
+---
+
+# `disconnect` Block
+
+<Placement groups={['job', 'group', 'disconnect']} />
+
+The `disconnect` block describes the systm's behaviour in case of a network 
+partition.
+
+```hcl
+    job "docs" {
+  	    group "example" {
+    	    disconnect {
+		        lost_after = "6h"
+		        stop_after = "2h"
+		        replace = false
+                reconcile = "keep_original"
+     	    }
+  	    }
+    }
+
+```
+
+## `disconnect` Parameters
+
+- `lost_after` `(string: "")` - Specifies a duration during which a Nomad client
+  will attempt to reconnect allocations after it fails to heartbeat
+  in the [`heartbeat_grace`] window.  It defaults to “” which is equivalent to 
```

Comment:
Some syntax highlighting of markdown will only pick up this as a link if you have a trailing empty bracket here. It's not required for rendering, but it's a nice-to-have:

```suggestion
  in the [`heartbeat_grace`][] window.  It defaults to "" which is equivalent to 
```